CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+
+# PayPal iOS SDK Release Notes
+
+## unreleased
+
+* Card
+  * Add `Address`
+  * Add `Card`
+  * Add `CardClient`
+  * Add `CardRequest`
+  * Add `CardResult`
+* Core
+  * Add `APIClient`
+  * Add `APIRequest`
+  * Add `AccessTokenRequest`
+  * Add `AccessTokenResponse`
+  * Add `CoreConfig`
+  * Add `Environment`
+  * Add `HTTPHeader`
+  * Add `HTTPMethod`
+  * Add `PayPalSDKError`
+  
```

Comment:
I left out the PayPal module for now since some of the interface is in flux with the web fallback. I also imagine we'll clean up the CHANGELOG before the first GA version.

Reply:
Are all these Core classes meant to be public facing?

Reply:
They are public because we need them to be used across module, but I don't think we need to call them out as public in our changelog.

Reply:
Got it yeah I can remove the Core piece from here for now. We can re-evaluate which Core classes should be advertised before GA for now this helps the script generate release notes.